web/src/Pages/TopicPage/TopicCard/TopicCard.test.js

```diff
@@ -20,14 +20,15 @@ describe("Tests for TopicCard using RTL", () => {
     expect(button.textContent).toMatch(/Tell me more/i);
     expect(image.src).toBe(topic.image);
   });
-  test("Renders dummy image if dummy data has no image", () => {
+  test("Renders dummy image if dummy data has no image", async() => {
```

Comment:
Async can be removed since we removed `await for` from the code below.
```suggestion
  test("Renders dummy image if dummy data has no image", () => {
```